activesupport/lib/active_support/logger_thread_safe_level.rb

```diff
@@ -28,7 +29,7 @@ def after_initialize
     end
 
     def local_log_id
-      Thread.current.__id__
```

Comment:
Should we rename it to more general one?
`ActiveSupport::LoggerSafeLocalLevel`

Reply:
No need to rename this module.